src/platforms/salesforce/client.ts

```diff
@@ -340,6 +357,52 @@ export const client = {
       schema: z.undefined(),
     })),
   },
+  contentNotes: {
+    find: query.find<typeof salesforceContentNote>({
+      objectType: 'ContentNote',
+      schema: salesforceContentNote,
+      relationalSelect: salesforceContentNoteRelationalSelect,
+    }),
+    list: query.list<typeof salesforceContentNote>({
+      objectType: 'ContentNote',
+      schema: salesforceContentNote,
+      relationalSelect: salesforceContentNoteRelationalSelect,
+    }),
+    create: request(({ ContentNote }: SalesforceContentNoteCreate) => ({
+      url: `/sobjects/ContentNote`,
+      method: 'POST',
+      json: ContentNote,
+      schema: salesforceContentNoteCreateResponse,
+    })),
+    update: request(({ Id, ContentNote }: SalesforceContentNoteUpdate) => ({
+      url: `/sobjects/ContentNote/${Id}/`,
+      method: 'PATCH',
+      json: ContentNote,
+      schema: salesforceContentNote,
+    })),
+    delete: request(({ Id }: { Id: string }) => ({
+      url: `/sobjects/ContentNote/${Id}/`,
+      method: 'DELETE',
+      schema: z.undefined(),
+    })),
+    contentBody: request(
+      ({ ContentNote }: { ContentNote: SalesforceContentNote }) => ({
```

Comment:
maybe we should just pass in content here? Wondering if it's confusing to take in the whole Note since this is the client? Would make more sense imo for the action

Reply:
Yeah, that makes sense, I'll just pass in the Content then